app/login/page.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
+  await page.goto('/login');
+
+  await page.getByTestId('email').click();
```

Comment:
Q: Is there a reason you didn't save `await page.getByTestId('email')` to it's own variable so you could just call that variable for your tests?

Reply:
Considering it's only called twice I didn't think to, but I can to reduce code. I'll make an update and push it.

Reply:
I tried this in multiple different ways and it was just _not_ having it. If @shashilo has a better idea I'm all ears.